build/webpack.base.conf.js

```diff
@@ -25,13 +25,13 @@ module.exports = {
   entry: {
     app: [
         'core-js/fn/promise', // required by the webpack runtime for async import(). babel polyfills don't help us here. ie11
-        './src/main.js'
+        './src/main.' + (process.env.NODE_ENV === 'module' ? 'vue' : 'js')
```

Comment:
`(process.env.NODE_ENV === 'module' ? 'vue' : 'js')` could do with being moved to the top under `const outputExt = (process.env.NODE_ENV === 'module' ? 'vue' : 'js');`  to make this config section clearer to read

Reply:
done. (except i used "inputExt" instead of "outputExt" since this file is a source file, not an output file.